### PR TITLE
Feat/ds 812 twig input props

### PR DESCRIPTION
**Pull request by literat**
Allow transfer of attributes directly to input/textarea element. Needed for usage with form-validations package where data attributes are used for custom messages when validating.

### CODE REVIEW

File: packages/web-twig/tests/components-fixtures/textFieldDefault.twig [hunk 1]
```diff
@@ -1,3 +1,5 @@
+{% set inputProps = {"data-validate": "true"} %}
```
Comment (crishpeen) [inline packages/web-twig/tests/components-fixtures/textFieldDefault.twig:1]:
```suggestion
{% set inputProps = { "data-validate": "true" } %}
```
everywhere? This looks nicer and we do it like this everywhere.

File: packages/web-twig/src/Resources/components/TextFieldBase/README.md [hunk 2]
```diff
@@ -65,6 +65,7 @@ Without lexer:
 | `helperText`        | `string`                                                             | `null`  | no       | Custom helper text                                                                                                                 |
 | `UNSAFE_helperText` | `string`                                                             | `null`  | no       | Unescaped custom helper text                                                                                                       |
 | `id`                | `string`                                                             | —       | yes      | Input and label identification                                                                                                     |
+| `inputProps`        | `string[]`                                                           | `[]`    | no       | Pass additional attributes for the input/textarea element                                                                          |
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/TextFieldBase/README.md:68]:
```suggestion
| `inputProps`        | `string[]`                                                           | `[]`    | no       | Pass additional attributes to the input/textarea element                                                                          |
```

This sound better to me (and to ai assistant 😁 )

**Review by crishpeen — APPROVED** [review]

File: packages/web-twig/src/Resources/components/TextFieldBase/README.md (hunk 2 above)
Comment (crishpeen) [reply packages/web-twig/src/Resources/components/TextFieldBase/README.md:68]: Also please if you decide to change it, do it everywhere.

File: packages/web-twig/tests/components-fixtures/textFieldDefault.twig (hunk 1 above)
Comment (literat) [reply packages/web-twig/tests/components-fixtures/textFieldDefault.twig]: Ok, fixed.

File: packages/web-twig/src/Resources/components/TextFieldBase/README.md (hunk 2 above)
Comment (literat) [reply packages/web-twig/src/Resources/components/TextFieldBase/README.md]: Ok, I changed this on places where I add this description.

**Pull request merged**